x-pack/plugins/observability/server/plugin.ts

```diff
@@ -105,7 +105,13 @@ export class ObservabilityPlugin implements Plugin<ObservabilityPluginSetup> {
         return coreStart.elasticsearch.client.asInternalUser;
       },
       ready: () => Promise.resolve(),
+      // For the line below this comment...
+      // so just .alerts? That doesn't seem right...
+      // I'm imagining this should be .alerts-observability and so
+      // ...ruleDataService.getFullAssetName('observability');
+      // otherwise .alerts could return top alerts for everything?
       alias: plugins.ruleRegistry.ruleDataService.getFullAssetName(),
+      feature: 'observability',
```

Comment:
What does listing the feature do here?

Reply:
The purpose of the `feature` param is to force the user to update the data structure which contains the mapping of consumers to alerts as data indices. The idea is it is typed such that it forces the user to go to the code and modify it. At least until a better system is put in place or we move the alerts as data client out of rule registry. 

https://github.com/dhurley14/kibana/pull/20/files#diff-84ed52fe42ac2740532045f107557a7f0f4d7a4ddd80fae202d12caf1edbdae0R39-R47